pkg/dns/proto/nds.pb.go

```diff
@@ -14,7 +14,7 @@
 
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.36.5
+// 	protoc-gen-go v1.36.6
 // 	protoc        (unknown)
 // source: dns/proto/nds.proto
```

Comment:
why did these change, but the source file did not?

Reply:
The tools dependency from when the branch was created vs when I ran the command more recently